docs/dictionary.rst

```diff
@@ -0,0 +1,1101 @@
+
+Further reading: `<https://datascience.nih.gov/nih-cloud-platform-interoperability-effort>`_  
+
+.. updated 2022-05-16  
+
+
+
+.. _dict Nextflow:
+
+Nextflow
+--------
+	A Java-based computational workflow engine. Dockstore supports the hosting of Nextflow workflows.  
+
+Further reading: `<https://www.nextflow.io/>`_  
+
+.. updated 2022-05-16  
+
+
+
+.. _dict NFL:
+
+NFL
+---
+*abbreviation for* :ref:`dict Nextflow`  
+
+	An uncommon acronym for :ref:`dict Nextflow`. This abbreviation is not used as frequently as :ref:`dict CWL` or :ref:`dict WDL`, but does see usage occasionally.  
+
+
+.. updated 2022-05-16  
+
+
+
+.. _dict NHGRI:
+
+NHGRI
+-----
+*abbreviation for* National Human Genome Research Institute  
+
+	A division of the :ref:`dict NIH` that focus on genomics research. Funds the :ref:`dict AnVIL Project`.  
+
+Further reading: `<https://www.genome.gov/>`_  
+
+.. updated 2022-05-16  
+
+
+
+.. _dict NHLBI:
+
+NHLBI
+-----
+*abbreviation for* National Heart, Lungs, and Blood Institute  
+
+	A division of the :ref:`dict NIH` that focuses on heart, lung, blood, and sleep health. Funds the :ref:`dict BioData Catalyst` platform.  
+
+Further reading: `<https://www.nhlbi.nih.gov/>`_  
+
+.. updated 2022-05-16  
+
+
+
+.. _dict NIH:
+
+NIH
+---
+*abbreviation for* National Institute of Health  
+
+	An American government institution, part of the Department of Health and Human Services (HHS), that engages in medical research.  
+
+Further reading: `<https://www.nih.gov/>`_  
+
+.. updated 2022-05-16  
+
+
+
+.. _dict OICR:
+
+OICR
+----
+*abbreviation for* Ontario Institute for Cancer Research  
+
+	A non-profit research institute based in Toronto that is focused on cancer detection and treatment. One of the two institutes involved in the development of Dockstore, the other being :ref:`dict UCSC`.  
+
+Further reading: `<https://oicr.on.ca/>`_  
+
+.. updated 2022-05-16  
+
+
+
+.. _dict ORCID:
+
+ORCID
+-----
+pronounced "or-kid", rhymes with kid  
+
+*abbreviation for* Open Researcher and Contributor ID  
+
+	ID used to identify researchers and their work in a way that doesn't solely rely on names.  
+
+Further reading: `<https://info.orcid.org/what-is-orcid/>`_  
+
+.. updated 2022-05-16  
+
+
+
+.. _dict organization:
+
+organization
+------------
+	In the context of Dockstore, an organization is a representation of some sort of institute, grant, project, or company. Organizations are created by Dockstore admins, but any user with at least two external accounts linked to their Dockstore account can request the creation of an organization on Dockstore.  
+
+Further reading: `<https://dockstore.org/organizations>`_  
+
+.. updated 2022-05-16  
+
+
+
+.. _dict parameter file:
+
+parameter file
+--------------
+	A :ref:`dict JSON` or :ref:`dict YAML` file that describes the inputs to a workflow. This usually includes internal links, or links to data in a Google or S3 bucket.  
```

Comment:
What is an internal link?

Reply:
I meant internal to the repo. How's this instead?
> A [JSON] or [YAML] file that describes the inputs to a workflow. This usually includes internal links to files within the same source-control repository, or links to data in a Google or S3 bucket.

Reply:
I would say relative or local links in that case. But:

1. There are also non-file links in there
2. You typically wouldn't want local/relative links if you were going to run it on something like Terra.

So I would suggest just dropping the internal/relative/local links section, and have something like "such as links to cloud data and runtime parameters".